cli/pipeline/run.go

```diff
@@ -145,6 +145,11 @@ func runPipelineStep(pipeline *PipelineDefinition, step *PipelineDefinitionStep,
 		return err
 	}
 
+	var gracePeriod int64
+	opts := metav1.DeleteOptions{}
+	opts.GracePeriodSeconds = &gracePeriod
+	pods.Delete(podDefinition.PodName, &opts)
```

Comment:
What happens if pod does not exist? Is there an error? Can we catch for that error in specific?